Flatbuffers 2.0.7 uses capital 'B' in 'FlatBuffersConfig.cmake'
We want to use our pybind11 instead of the one in git submodules
Author: Petr Král (INUITS)

Updated for 0.3.10 by Åke Sandgren, 2024-04-03
diff --git a/CMakeLists.txt b/CMakeLists.txt
index ea58278..c3bf295 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,7 +87,6 @@ if (BUILD_PYTHON_WHEEL)
         set(PYTHON_VERSION "")
     endif()
     find_package (Python ${PYTHON_VERSION} EXACT COMPONENTS Interpreter Development)
-    add_subdirectory(third_party/pybind11)
 endif()
 
 add_subdirectory(c++)
diff --git a/c++/CMakeLists.txt b/c++/CMakeLists.txt
index 05c1c1b..6efb894 100644
--- a/c++/CMakeLists.txt
+++ b/c++/CMakeLists.txt
@@ -17,7 +17,7 @@ else()
         COMPONENTS
             headers
     )
-    find_package(Flatbuffers REQUIRED)
+    find_package(FlatBuffers REQUIRED)
     find_package(zstd REQUIRED)
     find_package(ZLIB REQUIRED)
 
@@ -27,6 +27,7 @@ else()
 endif()
 
 find_package(Threads REQUIRED)
+find_package(pybind11 REQUIRED)
 
 find_program(
     FLATBUFFERS_FLATC_EXECUTABLE
